### DISCUSSION TITLE
A list of excellent ideas

### DISCUSSION THREAD

**Discussion by ghost** [WolfDen133/WFT-OLD#28] [Ideas]
So I have a few ideas that would work great!

1. Adding more tags (especially the player's username)
2. Adding a feature to enlarge the text because at some point or another we would like bigger texts to read from a distance.
3. Allowing more character inputs in form UI. Im not sure if its possible, but if not, maybe allow people to use /ft create <message> instead of going into UI.

**Comment by WolfDen133** [comment]
Hey, thanks for the input.

Currently due to the way that Minecraft works, you cannot enlarge texts to what ever size you want, becuase they are based of the nametags of entity's, even if you can change the scale of an entity, the actual nametag does not physically change size.

As for the more tags, I will be making a new floating text plugin with a better coded core, but that isn't going to be for another little while. 

And you can use the command to create texts without the use of the UI, do /ft help for a full guide on how to use the commands.

Thanks, WolfDen133.
